app/views/users/change_username.html.erb

```diff
@@ -1,12 +1,23 @@
 <!--Descriptive page name, messages and instructions-->
 <h2 class="heading"><%= ts("Change My User Name") %></h2>
 <%= error_messages_for :user %>
-<p class="caution">
-  <%= ts("Please use this feature with caution. For information on how changing your user name will affect your account, please check out the %{account_faq}. Note that changes to your username may take several days or longer to appear. If you are still seeing your old user name on your works, bookmarks, series, or collections after a week, please %{contact_support}.",
-  account_faq: (link_to ts("Account FAQ"), archive_faqs_path + "/your-account#namechange"),
-  contact_support: (link_to ts("contact Support"), new_feedback_report_path)
-  ).html_safe %>
-</p>
+<div class="caution">
+  <p>
+    <strong><%= t("users.change_username.caution") %></strong>
+    <%= t("users.change_username.change_window",
```

Comment:
In general we try to avoid concatenation when setting up I18n keys. In this case, the way "every X days" gets translated will be dependent on the language (e.g. because of plural rules) as well as the sentence containing it. The same applies to the "time ago" (e.g. [German](https://stackoverflow.com/questions/53013887/rails-is-there-a-built-translation-for-ago#comment113736195_53014319)) string on the inbox comments.

So I think we should:

- Set up the I18n keys for full sentences, like this (we skip "x times" as well since we're getting rid of `USER_RENAME_LIMIT`):
  ```yml
  change_window:
    one: "You can change your user name once every day."
    other: "You can change your user name once every %{count} days."
  ```
- Leave the custom helpers in `app/helpers/translation_helper.rb` and the inbox comment view alone for now.